tests/drivers/uart/uart_async_api/src/test_uart_async.c

```diff
@@ -366,6 +370,15 @@ void test_read_abort_callback(const struct device *dev,
 	}
 }
 
+static void read_abort_timeout(struct k_timer *timer)
+{
+	int err;
+
+	k_sem_reset(&rx_disabled);
```

Comment:
This shouldn't be needed. It is done in `test_read_abort_setup()`. For consistency, I think it would be better to remove this from here.